### DISCUSSION TITLE
Curl reports excess content found. Normal?

### DISCUSSION THREAD

**Discussion by ddevienne** [uNetworking/uWebSockets#1681] [Q&A] [answered]
The default built-in response when not finding a route lacks a `Content-Length` header,
yet seems to have content, which CURL reports as _excess_. Might need fixing? Just FYI.

I probably need to have a _catch all_ wildcard route, and report my own error I guess.

```
C:\Users\ddevienne>curl -v -I "http://localhost:8899/api/v1/acme"
*   Trying 127.0.0.1:8899...
* Connected to localhost (127.0.0.1) port 8899 (#0)
> HEAD /api/v1/acme HTTP/1.1
> Host: localhost:8899
> User-Agent: curl/8.0.1
> Accept: */*
>
< HTTP/1.1 404 File Not Found
HTTP/1.1 404 File Not Found

<
* Excess found: excess = 55 url = /api/v1/acme (zero-length body)
* Connection #0 to host localhost left intact
```

**Comment by uNetworkingAB** [comment, marked as answer]
https://github.com/uNetworking/uWebSockets/commit/c5aab195bd519dbe080d55092a23ca64e06325d8

this should fix it

**Reply by ddevienne** [reply]
Seems a bit severe to close the connection, although I kinda see your point, and you know best. Thanks.

**Reply by uNetworkingAB** [reply]
ideally 404 should not close connection but all other errors should. 404 needs to move somewhere else
